owl/EASE-OBJ.owl

```diff
@@ -1373,6 +1396,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#DependentPlace -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#DependentPlace">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Feature"/>
+        <rdfs:comment>A feature that is not part of its host, like a hole in a piece of cheese.</rdfs:comment>
```

Comment:
Oh I see, so it's not a part because it can't be taken out and have an independent existence. That's ok.